### PR TITLE
Building and running DuckDB on Windows 

### PR DESCRIPTION
**Pull request by szarnyasg**
Inspired by https://github.com/duckdb/duckdb/issues/8101

### CODE REVIEW

**Comment by szarnyasg** [comment]
@carlopi @Mause if you have any suggestions, please let me know

File: docs/api/python/known_issues.md [hunk 1]
```diff
@@ -48,3 +48,17 @@ In [2]: print(duckdb.sql("SELECT 42 AS x").explain())
...
+## Error When Importing the DuckDB Python Package on Windows
+
+When importing DuckDB on Windows, the Python runtime may return the following error:
+
+```python
+import duckdb
+```
+```
+>     from duckdb.duckdb.functional import (
+> ImportError: DLL load failed while importing duckdb: The specified module could not be found.
```
Comment (Mause) [inline docs/api/python/known_issues.md:61]:
```suggestion
ImportError: DLL load failed while importing duckdb: The specified module could not be found.
```

File: docs/api/python/known_issues.md [hunk 2]
```diff
@@ -48,3 +48,17 @@ In [2]: print(duckdb.sql("SELECT 42 AS x").explain())
...
+
+## Error When Importing the DuckDB Python Package on Windows
+
+When importing DuckDB on Windows, the Python runtime may return the following error:
+
+```python
+import duckdb
+```
+```
+>     from duckdb.duckdb.functional import (
```
Comment (Mause) [inline docs/api/python/known_issues.md:60]:
```suggestion
```

File: docs/api/python/known_issues.md [hunk 3]
```diff
@@ -48,3 +48,17 @@ In [2]: print(duckdb.sql("SELECT 42 AS x").explain())
...
 Please also check out the [Jupyter guide](../../guides/python/jupyter) for tips on using Jupyter with JupySQL.
+
+## Error When Importing the DuckDB Python Package on Windows
+
+When importing DuckDB on Windows, the Python runtime may return the following error:
+
+```python
+import duckdb
+```
+```
```
Comment (Mause) [inline docs/api/python/known_issues.md:59]:
```suggestion
```text
```

File: dev/building.md [hunk 4]
```diff
@@ -220,6 +220,29 @@ This can however hide include bugs, this flag disables using the unity build so
 In some situations, running an executable that has been built with sanitizers enabled is not support / can cause problems. Julia is an example of this.  
 With this flag enabled, the sanitizers are disabled for the build.
 
+## Dependencies of the Python Library on Windows
+
+The DuckDB dynamic library used on Windows (`.pyd`) has the following dependencies when built with Python 3.10:
```
Comment (Mause) [inline dev/building.md:225]: Listing out each ddl isn't really useful, just say that the visual C++ redistributable is required

**Comment by szarnyasg** [comment]
@Mause thanks

File: docs/api/python/known_issues.md [hunk 5]
```diff
@@ -48,3 +48,17 @@ In [2]: print(duckdb.sql("SELECT 42 AS x").explain())
...
+
+## Error When Importing the DuckDB Python Package on Windows
+
+When importing DuckDB on Windows, the Python runtime may return the following error:
+
+```python
+import duckdb
+```
+```text
+from duckdb.duckdb.functional import (
```
Comment (Mause) [inline docs/api/python/known_issues.md:60]: I think just drop this line, it isn't useful

**Review by Mause — DISMISSED** [review]

**Pull request merged**